events/tests/test_event_auth_api.py

```diff
@@ -370,17 +370,6 @@ def test_random_user_get_draft_event_not_found(self):
         response = self.client.get(url)
         self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
 
-    def test_random_user_create_event_denied(self):
```

Comment:
I'm not saying that this test shouldn't be deleted. Just thought that something similar could be tested when adding the setting `settings.ENABLE_EXTERNAL_USER_EVENT_CREATE`. So a test making sure that disabling `settings.ENABLE_EXTERNAL_USER_EVENT_CREATE` will work as intended.

Reply:
Added three new tests in `test_external_user.py` for create/update/delete when the flag is set to `False`.